chimg/chroot.py

```diff
@@ -14,6 +15,7 @@
 import yaml
 
 from chimg.common import run_command
+from chimg.config import ConfigFile
```

Comment:
don't import config. This should go through Context. Context should be the only direct user of Config. so use `self.ctx.conf`